pilot/pkg/bootstrap/server.go

```diff
@@ -121,13 +120,14 @@ type Server struct {
 	clusterID   string
 	environment *model.Environment
 
-	kubeConfig   *rest.Config
+	kubeConfig  *rest.Config
+	kubeClients kubelib.Client
+	// DEPRECATED. Use kubeClients
```

Comment:
is it too much to remove this now? Its awkward to have both .. should just be `kubeClient`.

Reply:
Yeah this is used in 26 different files. Ill do it in a followup PR?